esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -1977,6 +1977,51 @@ void CWsDfuEx::getFilePartsOnClusters(IEspContext &context, const char* clusterR
     }
 }
 
+void CWsDfuEx::parseFieldMask(unsigned __int64 fieldMask, unsigned &fieldCount, IntArray &fieldIndexArray)
+{
+    if (fieldMask == 0)
+        return;
+
+    fieldCount++;
+    if (fieldMask % 2)
```

Comment:
Not sure if this is the most efficient way, but probably doesn't make much difference.